libraries/core-react/stories/components/hooks/toCellValues.ts

```diff
@@ -0,0 +1,10 @@
+import { Data, Column } from '../helpers/data'
+
+export const toCellValues = (data: Data[], columns: Column[]): string[][] =>
+  data.map((item) =>
+    columns.map((column) =>
+      typeof item[column.accessor] !== 'undefined'
+        ? (item[column.accessor] as string)
+        : '',
+    ),
+  )
```

Comment:
Why was this moved to `hooks` ? Its not a hook and its very specific to made up example of how a `Table`  _could_ be done.

Reply:
I needed it for `Tooltip` in Table Cells example, but will move it to `helpers` instead  👍